cmd/batch-rotate.go

```diff
@@ -277,22 +277,22 @@ func (r *BatchJobKeyRotateV1) Start(ctx context.Context, api ObjectLayer, job Ba
 	skip := func(info FileInfo) (ok bool) {
 		if r.Flags.Filter.OlderThan > 0 && time.Since(info.ModTime) < r.Flags.Filter.OlderThan {
 			// skip all objects that are newer than specified older duration
-			return false
+			return true
```

Comment:
skip was written this way to filter by more than one criteria - e.g. metadata match and creation date criteria, which will not work with this change prematurely matching just one.  Perhaps just invert previous when passing to Walk Filter opts?